### ISSUE TITLE
resize images issues

### ISSUE THREAD

**Issue by hujiajia0401** [XiaoxiaoGuo/fashion-iq#4]
i have download the image using image url。subsequent， should i seize the image before the torchvision transforms？ cause some image size is less  than the crop size, and causing transfom
error。or should i change the crop size more lesser？

**Comment by hujiajia0401** [comment]
sorry ,i see the resize_image.py script

**Comment by zxh19976666** [comment]
How to divide the datasets into training set and test set，I have download all images，but how to divide it？
